mysql-test/suite/rpl/t/optimized_replace_mixed_engine.test

```diff
@@ -0,0 +1,55 @@
+--source include/have_log_bin.inc
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+--source include/have_debug.inc
+--source include/have_rocksdb.inc
+
+# Create a table in master with rocksdb as storage engine
+--echo Create a table in master with rocskdb as storage engine
+connection master;
+create table t1 (c1 int primary key, c2 int) engine=rocksdb;
+select table_name, engine from information_schema.tables where table_name = 't1';
+sync_slave_with_master;
```

Comment:
`sync_slave_with_master` is not considered reliable use the following include file instead
```suggestion
--source include/sync_slave_sql_with_master.inc
```